sdk/python/feast/feature_server.py

```diff
@@ -202,24 +202,27 @@ def materialize_incremental(body=Depends(get_body)):
     return app
 
 
-class FeastServeApplication(gunicorn.app.base.BaseApplication):
-    def __init__(self, store: "feast.FeatureStore", **options):
-        self._app = get_app(
-            store=store,
-            registry_ttl_sec=options.get("registry_ttl_sec", 5),
-        )
-        self._options = options
-        super().__init__()
+if sys.platform != "win32":
+    import gunicorn.app.base
 
-    def load_config(self):
-        for key, value in self._options.items():
-            if key.lower() in self.cfg.settings and value is not None:
-                self.cfg.set(key.lower(), value)
+    class FeastServeApplication(gunicorn.app.base.BaseApplication):
+        def __init__(self, store: "feast.FeatureStore", **options):
+            self._app = get_app(
+                store=store,
+                registry_ttl_sec=options.get("registry_ttl_sec", 5),
```

Comment:
nit: this should probably a config parameter somewhere up top

Reply:
fixed